server/server.go

```diff
@@ -968,45 +866,28 @@ func (s *server) SendOutboundMessage(ctx context.Context, req *rpcpb.SendOutboun
 	return &rpcpb.SendOutboundMessageResponse{Sent: sent}, err
 }
 
-func (s *server) LoadSnapshot(ctx context.Context, req *rpcpb.LoadSnapshotRequest) (*rpcpb.LoadSnapshotResponse, error) {
-	// if timeout is too small or not set, default to 5-min
-	if deadline, ok := ctx.Deadline(); !ok || time.Until(deadline) < defaultStartTimeout {
-		var cancel context.CancelFunc
-		ctx, cancel = context.WithTimeout(context.Background(), defaultStartTimeout)
-		_ = cancel // don't call since "start" is async, "curl" may not specify timeout
-		s.log.Info("received start request with default timeout", zap.String("timeout", defaultStartTimeout.String()))
-	} else {
-		s.log.Info("received start request with existing timeout", zap.String("timeout", deadline.String()))
-	}
-
-	// If [network] is already populated, the network has already been started.
-	if s.getNetwork() != nil {
-		return nil, ErrAlreadyBootstrapped
-	}
-
+func (s *server) LoadSnapshot(_ context.Context, req *rpcpb.LoadSnapshotRequest) (*rpcpb.LoadSnapshotResponse, error) {
```

Comment:
It isn't used in this method -- this method is non-blocking

Reply:
ok... I understood that maybe the ANR client will want to provide a timeout on the async op. is that reasonable?

Reply:
We can't use the context passed into `LoadSnapshot` for the async operations in the goroutine started by this method because the `ctx` will probably be cancelled after `LoadSnapshot` returns, which will kill the async goroutine.

Namely, the caller is probably doing something like:

```go
ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
result, err := server.LoadSnapshot(ctx, req)
cancel() // This kills the async goroutine even though we didn't mean to
```